.github/workflows/api-code-coverage.yml

```diff
@@ -0,0 +1,48 @@
+name: api-code-coverage
+on:
+  pull_request:
+    branches:
+      - development
+    paths:
+      - "backend/**/*"
+      - "!backend/.eslintrc.json"
+      - "!openshift/templates/api/docker/Dockerfile*"
+      - "!openshift/templates/api/docker/.dockerignore"
+      - "!backend/docs"
+      - "!backend/README.md"
+
+  workflow_dispatch:
+defaults:
+  run:
+    working-directory: backend/
+jobs:
+  api_lint_and_test_check:
+    if: github.repository_owner == 'bcgov'
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Caching node modules
+        uses: actions/cache@v3
+        env:
+          cache-name: cache-node-modules
+        with:
+          # npm cache files are stored in `~/.npm` on Linux/macOS
+          path: ~/.npm
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+
+      - name: Install packages
+        run: npm ci
+
+      - name: Run Backend Tests and Generate Coverage
+        run: |
```

Comment:
it might be nice to add the comment in the workflow where the code coverage percentage is set, otherwise it might not be obvious to look in the jest.config.js